test/backend/test_node.py

```diff
@@ -799,6 +799,34 @@ def test_onehot(self):
     output = run_node(node_def, inputs=[indices, depth, values])
     np.testing.assert_equal(output['y'], y)
 
+  def test_range(self):
+    if legacy_opset_pre_ver(11):
+      raise unittest.SkipTest("ONNX version {} doesn't support Range.".format(
+          defs.onnx_opset_version()))
+    node_def = helper.make_node(
+        "Range", ['start', 'limit', 'delta'], ['y'])
+    # test positive_delta
+    start = np.random.randint(0, 3, dtype=np.int32)
```

Comment:
Maybe create a class method, similar to _get_rnd, and call the class method instead of numpy so the coding style is consistent?

Reply:
Done. @chinchuang007 please review again.

Reply:
Since we simply wrap around np.random.randint, I recommend use the same default dtype as in https://docs.scipy.org/doc/numpy-1.15.0/reference/generated/numpy.random.randint.html, dtype=np.int. This part is pure numpy code, it feels strange to have dtype=tf.int8.

Reply:
You are right it should be np.int8, it is a typo.